conf/appliance_private_key

```diff
@@ -0,0 +1 @@
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMyPyer0kUFibsx1S4WLw8s+2SjIs/ZkeyCGiW/KQmqhZ0ZPj3ubeI6ljows/TsUSX3aeBaq2MK6+peA9THkFIA=
```

Comment:
I don't believe we want to add the appliance private key in the commit. Please remove this.